### ISSUE TITLE
Loading order issue result in null pointer (TaskManager)

### ISSUE THREAD

**Issue by Nils89** [Todoloo-WoW/Todoloo#54]
**Version**  
11.0.5.1

**Describe the bug**  
5x ...s/Todoloo/Source/Display/TaskTracker/TaskTracker.lua:50: attempt to index field 'TaskManager' (a nil value)
[string "@Todoloo/Source/Display/TaskTracker/TaskTracker.lua"]:50: in function `BuildGroupInfos'
[string "@Todoloo/Source/Display/TaskTracker/TaskTracker.lua"]:70: in function `EnumGroupData'
[string "@Todoloo/Source/Display/TaskTracker/TaskTracker.lua"]:233: in function `Update'
[string "@Todoloo/Source/Display/TaskTracker/FloatingTracker.lua"]:837: in function `TodolooTracker_Update'
[string "@Todoloo/Source/Display/TaskTracker/FloatingTracker.lua"]:695: in function <...doloo/Source/Display/TaskTracker/FloatingTracker.lua:689>

**To Reproduce**  
Just login on for me

I check the code and the Part where the Task manager get init is loaded ofter the first use and that result in this null pointer

**Comment by detilium** [comment]
Hey.

Thank you for the bug report.

Does this happen every time you login?

**Comment by Nils89** [comment]
Yes happens everytime. Maybe could also be a Mac client issue which was introduced with 11.0.5

**Comment by detilium** [comment]
Oh that may very well be. I haven't tested on Mac and I can't seem to replicate the issue on a Windows.

**Comment by Nils89** [comment]
Have just added the Task Manager Init above the error and and that solves the issue. Only if the Task Manager is Nil

**Comment by detilium** [comment]
You're welcome to make a PR if you've managed resolved the issue :)

**Comment by Nils89** [comment]
Sure, will do that next week :)

**Comment by detilium** [comment]
I'm about to release a new version in which the Todoloo table gets fully exposed for use in other addons.

If you can tell me exactly what you did and on what lines, I'd be able to include it in the release?
